### PR TITLE
[CIRCLE-37006] runner k8s cleanup

### PR DESCRIPTION
**Pull request by MaT1g3R**
- Remove runner on kubernetes preview note
- Move runner on k8s limitation to the bottom

# Reasons
JIRA: [CIRCLE-37006]

[CIRCLE-37006]: https://circleci.atlassian.net/browse/CIRCLE-37006

### CODE REVIEW

**Review by stalmy — APPROVED** [review]
thanks!

File: jekyll/_cci2/runner-on-kubernetes.adoc [hunk 1]
```diff
@@ -94,3 +87,8 @@ Customizable parameters are left inside the `+values.yaml+` file. See the follow
 
 |===
 
+=== Limitations/Pending Work
+* The Helm chart currently doesn't configure permissions, so the containers will not have elevated privileges. If you need elevated permissions (e.g., for Docker in Docker work, etc.), you will need to modify the chart yourself.
```
Comment (thomasr8) [inline jekyll/_cci2/runner-on-kubernetes.adoc:91]:
```suggestion
* The Helm chart currently does not configure permissions, therefore, the containers will not have elevated privileges. If you need elevated permissions (e.g., for Docker in Docker work, etc.), you will need to modify the chart yourself.
```

File: jekyll/_cci2/runner-on-kubernetes.adoc [hunk 2]
```diff
@@ -94,3 +87,8 @@ Customizable parameters are left inside the `+values.yaml+` file. See the follow
...
+* The Helm chart currently doesn't support auto-scaling - you will need to modify the `+replicaCount+` parameter inside `+values.yaml+` yourself and re-apply the chart to your cluster.
```
Comment (thomasr8) [inline jekyll/_cci2/runner-on-kubernetes.adoc:92]:
```suggestion
* The Helm chart currently does not support auto-scaling - you will need to modify the `+replicaCount+` parameter inside `+values.yaml+` yourself and re-apply the chart to your cluster.
```

File: jekyll/_cci2/runner-on-kubernetes.adoc [hunk 3]
```diff
@@ -94,3 +87,8 @@ Customizable parameters are left inside the `+values.yaml+` file. See the follow
...
+* The Helm chart currently expects only one runner resource class type and one token. If you want to configure clusters for multiple runner resource classes, you'll need to set up separate charts for each one.
```
Comment (thomasr8) [inline jekyll/_cci2/runner-on-kubernetes.adoc:93]:
```suggestion
* The Helm chart currently expects only one runner resource class type and one token. If you want to configure clusters for multiple runner resource classes, you will need to set up separate charts for each one.
```

File: jekyll/_cci2_ja/runner-on-kubernetes.adoc [hunk 4]
```diff
@@ -94,3 +87,8 @@ Customizable parameters are left inside the `+values.yaml+` file. See the follow
 
 |===
 
+=== Limitations/Pending Work
+* The Helm chart currently doesn't configure permissions, so the containers will not have elevated privileges. If you need elevated permissions (e.g., for Docker in Docker work, etc.), you will need to modify the chart yourself.
```
Comment (thomasr8) [inline jekyll/_cci2_ja/runner-on-kubernetes.adoc:91]:
```suggestion
* The Helm chart currently does not configure permissions, so the containers will not have elevated privileges. If you need elevated permissions (e.g., for Docker in Docker work, etc.), you will need to modify the chart yourself.
```

File: jekyll/_cci2_ja/runner-on-kubernetes.adoc [hunk 5]
```diff
@@ -94,3 +87,8 @@ Customizable parameters are left inside the `+values.yaml+` file. See the follow
...
+* The Helm chart currently doesn't support auto-scaling - you will need to modify the `+replicaCount+` parameter inside `+values.yaml+` yourself and re-apply the chart to your cluster.
```
Comment (thomasr8) [inline jekyll/_cci2_ja/runner-on-kubernetes.adoc:92]:
```suggestion
* The Helm chart currently does not support auto-scaling - you will need to modify the `+replicaCount+` parameter inside `+values.yaml+` yourself and re-apply the chart to your cluster.
```

**Review by thomasr8 — APPROVED** [review]
Just some very minor comments around contractions (we do not use them in our content because of localization and translation issues). Otherwise, this is really good. Thank You 👍

**Pull request merged**